examples/platform/efr32/MemMonitoring.h

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#ifdef HEAP_MONITORING
+#include "FreeRTOS.h"
+
+#define MONITORING_STACK_SIZE_byte 1024
+
+class MemMonitoring
```

Comment:
could we use a namespace here? Can the HeapMonitoring thread method be just defined in the cpp file?

Reply:
We don't use namespace in our example for the AppTask, I don't see any reason why we should use one since it's platform and exemple specific code...